src/lib/users/ssh_authorized_keyring.rb

```diff
@@ -105,7 +106,7 @@ def initialize(home)
       #
       # @return [Array<String>] Registered authorized keys
       def add_keys(new_keys)
-        keys.concat(new_keys)
+        @keys |= new_keys.compact
```

Comment:
really bitwise operator here? I expect `||=`

Reply:
Actually, I'm using it there as a "shorthand" of union

```
[I] ➜ irb
irb(main):001:0> a = [2, 3]
=> [2, 3]
irb(main):002:0> b = [4, 3]
=> [4, 3]
irb(main):003:0> a |= b
=> [2, 3, 4]
irb(main):004:0> a
=> [2, 3, 4]
irb(main):005:0> 
```

Reply:
nice trick